pkg/domain/infra/tunnel/generate.go

```diff
@@ -43,3 +44,7 @@ func (ic *ContainerEngine) GenerateKube(ctx context.Context, nameOrIDs []string,
 	options := new(generate.KubeOptions).WithService(opts.Service)
 	return generate.Kube(ic.ClientCtx, nameOrIDs, options)
 }
+
+func (ic *ContainerEngine) GenerateSpec(ctx context.Context, opts *entities.GenerateSpecOptions) (*entities.GenerateSpecReport, error) {
+	return nil, fmt.Errorf("GenerateSpec is not supported on the remote API")
```

Comment:
Reason for this?

Reply:
I mean, I could add a remote endpoint but does this have a usecase remotely at the moment since the only usage is really directly piping to the API?

Reply:
I think it actually makes a lot of sense that an API user would want to get the API definition of an existing container?